schema/release-schema.json

```
@@ -1006,6 +1006,11 @@
             "$ref": "#/definitions/Document"
           },
           "uniqueItems": true
+        },
+        "value": {
+          "title": "Value",
+          "description": "If the milestone represents a planned payment, the payment's value.",
```

Start with the semantics, then add qualifiers.

```suggestion
          "description": "The payment's value, if the milestone represents a planned payment.",
```